ui/framework/src/appui-react/frontstage/Frontstage.tsx

```diff
@@ -54,7 +54,8 @@ export interface FrontstageProps extends CommonProps {
    * Version increase is required when widgets are added/removed.
    */
   version?: number;
-
+  /** if isModelIndependent then frontstage is independent from any iModel. */
+  isModelIndependent?: boolean;
```

Comment:
Please call it `isIModelIndependent` - a "model" is different from an "iModel".

Reply:
good catch... I meant to fix that

Reply:
fixed